iree/compiler/Conversion/Common/LinalgBufferizePass.cpp

```diff
@@ -85,6 +89,25 @@ static Value createSubviewOp(OpBuilder &b, Location loc, Value src,
   return b.create<SubViewOp>(loc, src, offsets, sizes, strides);
 }
 
+//===----------------------------------------------------------------------===//
+// Bufferization helper functions for scalar operations.
+//===----------------------------------------------------------------------===//
+
+/// Converts a `tensor.extract` operation into a `load`.
+static LogicalResult convertTensorExtractOp(OpBuilder &b, tensor::ExtractOp op,
+                                            BlockAndValueMapping &bvm) {
+  OpBuilder::InsertionGuard g(b);
+  b.setInsertionPoint(op);
+  Value inputBuffer = bvm.lookup(op.tensor());
+  Value load = b.createOrFold<LoadOp>(op.getLoc(), inputBuffer, op.indices());
+  bvm.map(op.result(), load);
+  return success();
+}
```

Comment:
I feel the conflicts were fixed in a wrong way?

This should be moved to below. And the comment does not make sense?

Reply:
Sorry stale refactoring that wasnt needed. Moving it back.